www/source/partials/docs/_bp-kubernetes.html.md.erb

```diff
@@ -1,17 +1,29 @@
 # <a name="kubernetes" id="kubernetes" data-magellan-target="kubernetes">Kubernetes</a>
 [Kubernetes](http://kubernetes.io/) is an open source container cluster manager that is available as a stand-alone platform or embedded in several distributed platforms including [Google's Container Engine](https://cloud.google.com/container-engine/), [Tectonic](https://tectonic.com/) by [CoreOS](https://coreos.com/), and [OpenShift](https://openshift.com/) by [RedHat](https://redhat.com). Habitat and Kubernetes are complementary: Kubernetes focuses on providing a platform for deployment, scaling, and operations of application containers across clusters of hosts while Habitat manages the build pipeline and lifecycle of those application containers.
 
-## Habitat Kubernetes Operator
+## Habitat Operator
 
-The [Habitat Kubernetes Operator](https://github.com/kinvolk/habitat-operator) is on-going work to create an operator that leverages Kubernetes API services to create a native and robust integration between the two technologies. Follow along on [github](https://github.com/kinvolk/habitat-operator), and join us in our #kubernetes channel in the [Habitat Slack Channel](https://slack.habitat.sh) for more information.
+The [Habitat Kubernetes Operator](https://github.com/kinvolk/habitat-operator) is on-going work to create an operator that leverages Kubernetes API services to create a native and robust integration between the two technologies.
+
+By using the Habitat Operator, you can abstract from many of the low level details of running a Habitat package in Kubernetes, and jump straight to deploying your application, with support for features like [service configuration](https://github.com/kinvolk/habitat-operator/tree/master/examples/config), [binding](https://github.com/kinvolk/habitat-operator/tree/master/examples/bind), [topologies](https://github.com/kinvolk/habitat-operator/tree/master/examples/leader) and more.
```

Comment:
> with support for features like

instead I would say

> with support for Habitat features like

Reply:
Done.

BTW, diffs like this one are why I prefer having line breaks in Markdown files ;)